.circleci/config.yml

```diff
@@ -57,6 +57,30 @@ jobs:
                   else
                       echo "Latest version is already published."
                   fi
+            - run:
+                name: Publish @pxblue/react-native-template-routing
```

Comment:
Make sure to cd into the correct folder before you start publishing.

Reply:
good catch